app/jobs/leader_check_ins_job.rb

```diff
@@ -28,7 +28,7 @@ def perform(streak_keys = [])
 
   private
 
-  def retrieive_user_ids(streak_keys)
+  def retrieve_user_ids(streak_keys)
```

Comment:
Was this change intentional? Seems separate from the rest of the changes in this PR.